templates/contributor/show.html.twig

```diff
@@ -0,0 +1,45 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}{{ contributor.name }} Profil{% endblock %}
+
+{% block body %}
+    <div class="row">
+        <div class="container">
+            <div class="row row-cols-1">
+                <div class="col my-3 d-flex justify-content-between">
+                    {% include "component/_toast.html.twig" %}
+                    <a class="add-link link-dark" href="{{ path('contributor_edit', {id: contributor.id}) }}">Modify</a>
+                    <form method="POST" action="{{ path('contributor_delete', {'id': contributor.id}) }}"
+                          onsubmit="return confirm('Are you sure ?');">
+                        <input type="hidden" name="_token"
+                               value="{{ csrf_token('_delete' ~ contributor.id) }}">
+                        <button class="delete-project-button btn btn-outline-danger text-dark">Delete</button>
+                    </form>
+                </div>
+
+                <div class=" col bg-white border border-dark d-flex flex-row align-items-center justify-content-between align-content-center p-1 p-md-2 p-lg-4 my-2">
+                    <div class="d-flex flex-column">
+                        <h1 class="project-title align-self-center m-0">Name : </h1>
+                        <p class="project-name align-self-center">{{ contributor.name }}</p>
+                    </div>
+                    {% if contributor.projects is defined and contributor.projects is not null %}
```

Comment:
the "is defined" control is useless : it will call the getProjects() of a contributor so it will always be defined